how-to/web-layout/public/platform/provider.html

```diff
@@ -19,14 +19,17 @@ <h1>Web Multiple Layout Example</h1>
 				<h1 class="tag">Demonstrate web interop usage with multiple layouts</h1>
 			</div>
 			<div class="row middle gap20">
-				<button id="swap-layouts">Swap Layouts</button>
+				<button id="add-layout">+</button>
```

Comment:
If you want joe you can take the + SVG from the support context and intents example as it is a bit bigger.

Reply:
Couldn't find the svg, but added the word "Add" to make it more obvious